float64 stamp
string cmd
